roles/convert/README.md

```diff
@@ -1,38 +1,45 @@
-Role Name
+Convert
 =========
 
-A brief description of the role goes here.
+The `convert` role is used to perform the actual conversion of the target hosts. After the conversion is complete target hosts are cleaned up by removing the `convert2rhel` package and any related files.
 
-Requirements
-------------
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+This role will not fail if there are inhibitors found, it will throw a warning. However, there is a fact available `conversion_inhibited` which indicates whether the upgrade will be inhibited or not and you can choose to fail your own playbook using this variable.
```

Comment:
```suggestion
This role will not fail if there are inhibitors found, it will throw a warning. However, there is a fact available `conversion_inhibited` which indicates whether the conversion will be inhibited or not and you can choose to fail your own playbook using this variable.
```